angular/src/app/modules/branch-manager/project-management/project-management.component.ts

```diff
@@ -164,4 +179,54 @@ export class ProjectManagementComponent extends PagedListingComponentBase<any> i
     throw new Error('Method not implemented.');
   }
 
-}
+  customDate() {
+    if (this.typeOfView == this.APP_CONSTANT.TypeViewBranchManager.Day) {
```

Comment:
move to common service, and reuse in project-management.component and manage-employee.component